### PR TITLE
ZK-4075: Combobox browser inconsistent autocomplete behavior

### PR DESCRIPTION
**Pull request by charlesqqq**

### CODE REVIEW

File: zul/src/archive/web/js/zul/inp/Combobox.js [hunk 1]
```diff
@@ -445,15 +447,29 @@ zul.inp.Combobox = zk.$extends(zul.inp.ComboWidget, {
...
 		// Bug ZK-403
 		if (this.isListen('onOpen'))
 			this.unlisten({onChanging: zk.$void});
+		var input = this.getInputNode();
+		this.domUnlisten_(input, 'onCompositionend', '_doCompositionend')
+			.domUnlisten_(input, 'onCompositionstart', '_doCompositionstart');
 		this.$supers(zul.inp.Combobox, 'unbind_', arguments);
 	},
+	_doCompositionstart: function () {
+		this._suppressAutoComplete = true;
```
Comment (DevChu) [inline zul/src/archive/web/js/zul/inp/Combobox.js]: autoCompleteSuppressed

File: zul/src/archive/web/js/zul/inp/Combobox.js [hunk 2]
```diff
@@ -377,6 +377,8 @@ zul.inp.Combobox = zk.$extends(zul.inp.ComboWidget, {
 			}
 	},
 	doKeyUp_: function (evt) {
+		if (this._suppressAutoComplete)
+			return;
```
Comment (DevChu) [inline zul/src/archive/web/js/zul/inp/Combobox.js]: check super. doKeyUp_

File: zul/src/archive/web/js/zul/inp/Combobox.js [hunk 3]
```diff
@@ -445,15 +447,29 @@ zul.inp.Combobox = zk.$extends(zul.inp.ComboWidget, {
...
+		this.domListen_(input, 'onCompositionstart', '_doCompositionstart')
+			.domListen_(input, 'onCompositionend', '_doCompositionend');
 	},
 	unbind_: function () {
 		this._hilite2();
 		this._sel = this._lastsel = null;
 		// Bug ZK-403
 		if (this.isListen('onOpen'))
 			this.unlisten({onChanging: zk.$void});
+		var input = this.getInputNode();
```
Comment (DevChu) [inline zul/src/archive/web/js/zul/inp/Combobox.js]: order

**Comment by DevChu** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**